std/experimental/logger/core.d

```diff
@@ -1,17 +1,9 @@
 ///
 module std.experimental.logger.core;
 
-import std.array;
-import std.stdio;
-import std.conv;
 import std.datetime;
-import std.string;
-import std.range;
+import std.range.primitives;
 import std.traits;
```

Comment:
There is a benefit right now: the compiler has to search all the top-level modules and any publicly imported modules they reference to see where a symbol is invoked from and check for name collisions.  When you specify the module a symbol comes from, that search is skipped.

The real issue is that we have held off on using module-scope selective imports until we're sure the symbol leak bugs are fixed and downstream code has adapted.  Now that 2.073 has shipped, maybe @MartinNowak will OK their use?